script.js

```diff
@@ -0,0 +1,48 @@
+/*const inputAmountInput = document.querySelector(".input-input");
+const calcInput = document.querySelector(".calc-input");
+
+const inputAmountBut = document.querySelector(".input-but");
+let calcBut = document.querySelector(".calc-but");
+
+let riskInput = [];
+
+let canAdd = true;
+
+function checkInput() {
+  if (inputAmountInput.value >= riskInput.length) {
+    canAdd = false;
+}
+
+function addSum () {
+  
+}
+*/
+//const numberInputs = document.querySelectorAll(".input");
+const numInp1 = document.querySelector(".input1");
+const numInp2 = document.querySelector(".input2");
+const numInp3 = document.querySelector(".input3");
+const numInp4 = document.querySelector(".input4");
+const numInp5 = document.querySelector(".input5");
+const numInp6 = document.querySelector(".input6");
```

Comment:
Make sure to have default value as 0 in each textbox if user has not entered anything to avoid null pointer exceptions